src/Node/SandboxNode.php

```diff
@@ -34,12 +34,15 @@ public function compile(Compiler $compiler): void
             ->write("\$this->sandbox->enableSandbox();\n")
             ->outdent()
             ->write("}\n")
+            ->write("try {\n")
```

Comment:
to generate good looking code, please add the `->indent()` and `->outdent()` calls  to indent the generated code inside the `try` and `finally` blocks.

Reply:
Cool, thanks. Didn't knew that.